.github/workflows/cd.yaml

```
@@ -0,0 +1,82 @@
+name: Qt Project CD
+
+on:
+  push:
+    branches:
+      - ["**"]
```

actionlint (1.7.4)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)